index-handlers/src/main/java/no/sikt/nva/nvi/index/aws/OpenSearchClient.java

```diff
@@ -80,14 +80,14 @@ public void addDocumentToIndex(NviCandidateIndexDocument indexDocument) {
     }
 
     @Override
-    public void removeDocumentFromIndex(NviCandidateIndexDocument indexDocument) throws IOException {
-        client.withTransportOptions(getOptions()).delete(contructDeleteRequest(indexDocument));
+    public void removeDocumentFromIndex(NviCandidateIndexDocument indexDocument) {
+        attempt(() -> client.delete(contructDeleteRequest(indexDocument))).orElseThrow();
```

Comment:
Why have you removed `client.withTransportOptions(getOptions())`? Don't we need it for renewing jwt?